docs/en/02_Developer_Guides/14_Files/03_File_Security.md

```diff
@@ -9,7 +9,7 @@ in your system. As pages and dataobjects can be either versioned, or restricted
 members, it is necessary at times to apply similar logic to any files which are attached to these objects
 in the same way.
 
-Out of the box SilverStripe comes with an two asset stores, a public and a protected one.
+Out of the box, SilverStripe comes with an two asset stores: a public and a protected one.
```

Comment:
Out of the box, SilverStripe comes with ~an~ two asset stores: a public and a protected one.

Reply:
```suggestion
Out of the box, SilverStripe comes with two asset stores: a public and a protected one.
```